src/ai/backend/client/kernel.py

```diff
@@ -32,6 +32,7 @@ class BaseKernel(BaseFunction):
 
     @classmethod
     def _get_or_create(cls, lang: str,
+                       tag: str = None,
```

Comment:
The order of `get_or_create()` method matters currently (e.g., [here](https://github.com/lablup/neumann/blob/6e4edddf3/ingen/views.py#L313)), so please put the tag parameter back.
In the next version, all arguments except the first `lang` parameter will become keyword-only.